src/writeFile.js

```diff
@@ -1,4 +1,4 @@
-import fs from 'fs';
+import fs from 'fs-extra';
```

Comment:
`mkdirp` instead of the 'whole' `fs-extra` dependency?

Reply:
@michael-ciniawsky `mkdirp` have many issue with windows, `fs-extra` more stable

Reply:
kk